src/pages/contract/ExecuteContract.tsx

```diff
@@ -75,9 +86,7 @@ export function ExecuteContract({ contractAddress }: Props): JSX.Element {
           <JSONInput
             width="100%"
             height="200px"
-            placeholder={{
-              transfer: { recipient: "cosmos1zk4hr47hlch274x28j32dgnhuyewqjrwxn4mvm", amount: "1" },
-            }}
+            placeholder={executePlaceholder}
```

Comment:
Can we unset this for simplification? It should not be needed anymore, right?

Reply:
The placeholder is the text the user sees and can edit on the editor. If I remove the placehoder attribute the msgObject would be prefilled and you would be able to execute it, but you wouldn't see anything on the editor

Reply:
Thanks for clarification